tests/apps/courses/test_templates_course_detail_rdfa.py

```diff
@@ -523,6 +523,7 @@ def test_templates_course_detail_rdfa(self):
             self.assertTrue(subject in course_run_subjects)
 
         for end_date in ["2030-07-10", "2030-08-01"]:
+            subject, = graph.subjects(
```

Comment:
seems to break the build... keep it like it is?

Reply:
I didn't change this file... merge problem?

Reply:
It has take me some time, but I fixed. The branch add-schema-metadata-courses had different hashed between the upstream openfun and my fork. So at least now I don't have any conflicts.